pkg/proxy/apis/kubeproxyconfig/v1alpha1/types.go

```diff
@@ -22,15 +22,15 @@ import (
 
 // ClientConnectionConfiguration contains details for constructing a client.
 type ClientConnectionConfiguration struct {
-	// kubeConfigFile is the path to a kubeconfig file.
+	// kubeconfig is the path to a kubeconfig file.
 	KubeConfigFile string `json:"kubeconfig"`
```

Comment:
shouldn't we rename the field?

Reply:
If you want, we can. I think the json tag is most important, since that's user-facing.

Reply:
I think golang lint style requires the KubeConfigFile (capitalized) here unless you want to rename the var